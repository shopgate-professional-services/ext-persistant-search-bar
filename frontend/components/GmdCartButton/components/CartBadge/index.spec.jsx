import React from 'react';
import { mount } from 'enzyme';
import CartBadge from './index';

const badgeProps = {
  count: 1,
};

describe('<CartBadge />', () => {
  it('should render', () => {
    const component = mount(<CartBadge {...badgeProps} />);
    expect(component).toMatchSnapshot();
  });
});
