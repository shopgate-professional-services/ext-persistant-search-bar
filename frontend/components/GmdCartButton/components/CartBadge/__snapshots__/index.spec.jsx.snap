// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CartBadge /> should render 1`] = `
<CartButtonBadge
  count={1}
>
  <div
    className={
      Object {
        "data-css-7g35sj": "",
      }
    }
  >
    1
  </div>
</CartButtonBadge>
`;
